app/client/ui/UserManager.ts

```diff
@@ -518,90 +522,60 @@ export class UserManager extends Disposable {
   }
 }
 
+function getUserItem(member: IEditableMember): ACUserItem {
+  return {
+    value: member.email,
+    label: member.email,
+    cleanText: normalizeText(member.email),
+    email: member.email,
+    name: member.name,
+    picture: member?.picture,
+    id: member.id,
+  }
+}
+
 /**
  * Represents the widget that allows typing in an email and adding it.
- * The border of the input turns green when the email is considered valid.
  */
-export class MemberEmail extends Disposable {
+export class ACMemberEmail extends Disposable {
   public email = this.autoDispose(observable<string>(""));
```

Comment:
It seems ok to make this private.
```suggestion
  private _email = this.autoDispose(observable<string>(""));
```